src/applications/dependents/view-dependents/components/ViewDependentsHeader/ViewDependentsHeaderV2.jsx

```diff
@@ -17,22 +17,27 @@ const CALLSTATUS = {
 function ViewDependentsHeader(props) {
   const { updateDiariesStatus, showAlert } = props;
 
-  const [warningVisible, setWarningVisible] = useState(showAlert);
+  const [warningHidden, setWarningHidden] = useState(false);
 
-  useEffect(() => {
-    const handleWarningClose = () => setWarningVisible(false);
-
-    const warningAlert = document.getElementById('update-warning-alert');
-    if (warningAlert) {
-      warningAlert.addEventListener('closeEvent', handleWarningClose);
-    }
+  useEffect(
+    () => {
+      const handleWarningClose = () => {
+        setWarningHidden(true);
+      };
 
-    return () => {
+      const warningAlert = document.getElementById('update-warning-alert');
       if (warningAlert) {
-        warningAlert.removeEventListener('closeEvent', handleWarningClose);
+        warningAlert.addEventListener('closeEvent', handleWarningClose);
```

Comment:
I think this was written before web components. The `va-alert` does have an `closeEvent` event... add a callback using `onCloseEvent` instead of using an event listener.

Reply:
We also need to think about where to move focus once the alert is closed. I think the h2 (Dependents on your VA benefits) would be good.